camptix.php

```diff
@@ -2791,43 +2791,33 @@ function generate_attendee_report( $format ) {
 			$report = '<?xml version="1.0" encoding="UTF-8"?>' . PHP_EOL . '<attendees>' . PHP_EOL;
 
 		$paged = 1;
+		$buyer_map = array();
+
+		// Ordering by ID ASC is important. Presence of buyer row depends on it. Buyer has to fetched, same or before rest of attendees rows are fetched.
```

Comment:
Would it be good to add some kind of `group by tix_access_token` clause to ensure that related tickets are always in the same chunk? That's probably not possible w/ `get_posts`, but should be with a `WP_query` param.

Reply:
Can you elaborate, I am not sure what `group by tix_access_token` achieve. 

Do you mean `order by tix_access_token`? If so then probably that won;'t be a good idea, because meta_value column is not indexed and mysql will use a temporary table to compute the results, and this will be like really really slow. Also, because of how we create tickets, I guess order by ID should be good enough?

Reply:
Ah, yeah, good point :)